fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1357,9 +1357,7 @@ private void startAllNodeTypeDaemonThreads() {
         // ES state store
         esRepository.start();
 
-        if (Config.enable_hms_events_incremental_sync) {
-            metastoreEventsProcessor.start();
-        }
+        metastoreEventsProcessor.start();
```

Comment:
why you do this ? Please provide detailed reasons

Reply:
1 set CATALOG PROPERTIES enable_hms_events_incremental_sync=true for one catalog 
2 but  fe.conf param enable_hms_events_incremental_sync=false
3 then can’t enable automatic incremental update for a single Hive catalog.
<img width="1020" alt="image" src="https://github.com/StarRocks/starrocks/assets/13237066/0cdac456-7ca8-45b5-b7f7-746a1895f824">

Reply:
Please put the issue in pr body

Reply:
thanks for your contribution. the incremental refreshing based on the hms event is deprecated. I recommand that   you use https://docs.starrocks.io/docs/data_source/catalog/hive_catalog/#periodically-refresh-metadata-cache instread of incremental refreshing.

Reply:
tks review @stephen-shelby 
Should the Automatic incremental update doc removed?